pytorch_lightning/cluster_environments/cluster_environment.py

```diff
@@ -29,3 +32,23 @@ def world_size(self):
 
     def local_rank(self):
         pass
+
+    def required_plugins(self, amp_backend: AMPType) -> Optional[list]:
```

Comment:
shall it be rather always a list and in none, then empty?
it would be much easier later if you want to append something or combine together...